packages/@aws-cdk/aws-codepipeline/README.md

```diff
@@ -80,16 +96,91 @@ or you can use the `IStage.addAction()` method to mutate an existing Stage:
 sourceStage.addAction(someAction);
 ```
 
+### Cross-account CodePipelines
+
+Cross-account Pipeline actions require that the Pipeline has *not* been
+created with `crossAccountKeys: false`.
+
+To perform actions in a different account than your Pipeline is in, most
+actions accept a resource that is in a different account (either created
+or imported):
+
+```typescript
+stage.addAction(new codepipeline_actions.S3DeployAction({
+  bucket: s3.Bucket.fromBucketAttributes(this, 'Bucket', {
+    account: '123456789012',
+    // ...
+  }),
+  // ...
+}));
+```
+
+Some actions accept an explicit `account` parameter:
+
+```typescript
+stage.addAction(new codepipeline_actions.CloudFormationCreateUpdateStackAction({
+  account: '123456789012',
+  // ...
+}));
+```
+
+CodePipeline requires that an IAM Role exists in the target account with a
+well-known name. The `Pipeline` construct automatically defines a **support
+stack** for you, named `<PipelineStackName>-support-<account>`, that will
+provision a role that the pipeline will assume in the given account before
+executing this action. This support stack will automatically be deployed
+before the stack containing the pipeline.
```

Comment:
It would be nice to start the paragraph with something like this. And move what's above this paragraph after.
I don't mean exactly as is (of course), but content. 

More generically - start by stating the feature, then how to use it and then how it's implemented. The second and third can be slightly interleaved if it makes for easier reading.

Reply:
I'm having a hard time describing what the feature is without describing the context in which the feature operates (which is that we need a role and we either create one for you or you specify one).

Reply:
"Pipelines can define Actions that are performed in a different account. In order to execute this action in the target account, the `Pipeline` construct automatically defines an **IAM Role** for you in the target account which the pipeline will assume to perform that action...."

Why not start like this? Then say,

"Actions that can be deployed in a different account take the `accountId` as an option. S3DeployAction and CodeBuildAction are two such examples:

<code samples>
"